sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -338,6 +338,11 @@ export class OfferingValidationModel {
   /**
    * Actual tuition costs.
    */
+  @ValidateIf(
+    (offering: OfferingValidationModel) => {
+      return offering.actualTuitionCosts !== undefined
+    }
```

Comment:
It is not a blocker, it can be simplified as below. No need for a return and brackets.
```ts
(offering: OfferingValidationModel) => offering.actualTuitionCosts !== undefined
```

Reply:
updated